src/com/lftechnology/realworldassign/ProductServices.java

```diff
@@ -0,0 +1,52 @@
+package com.lftechnology.realworldassign;
+
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This class consists of reusable methods of the Product items such as remove and update of the items in the List.
+ * 
+ * @author bibhushan<bibhushanjoshi@lftechnology.com>
+ * 
+ */
+public class ProductServices {
+	private static final Logger LOGGER = Logger.getLogger(ProductServices.class.getName());
+
+	/**
+	 * This method removes {@link String} value from the {@link List}
+	 * 
+	 * @param inputList
+	 *            is {@link List} from which value is to be removed
+	 * @param value
+	 *            is{@link String} to be removed
+	 */
+	public static void removeItem(List<String> inputList, String value) {
+		if (inputList.contains(value)) {
+			inputList.remove(value);
+			LOGGER.log(Level.INFO, "removed value: {0}\n", value);
+		} else {
+			LOGGER.log(Level.WARNING, " value not present in the list : {0}\n", value);
+		}
+	}
+
+	/**
+	 * This method updates the {@link String} value in a {@link List}
+	 * 
+	 * @param inputList
+	 *            is a {@link List} in which value is to be updated.
+	 * @param oldValue
+	 *            is a {@link String} value which is to be changed with new value.
+	 * @param newValue
+	 *            is a {@link String} value which replaces old value.
+	 */
+	public static void updateItem(List<String> inputList, String oldValue, String newValue) {
```

Comment:
Is there a possibility of Duplicate values in List?  Do you need to consider duplicate values entries in List?